packages/web/components/complex/transfer.tsx

```diff
@@ -92,7 +92,8 @@ export const Transfer: FunctionComponent<TransferProps> = ({
       : isMobile
       ? 14
       : 24;
-
+  // Add query to congestion status here
+  const ibcCongestionStatus = useState(false);
```

Comment:
This comes with experience with the codebase- but this component is not just used to display IBC transfers, but also transfers from Ethereum via our Axelar integration (check out USDC or WETH transfer modals).

For that reason, I don't think we should put IBC-specific logic here, but instead we should add more generic capabilities to this component, or update & reuse the `warningMessage` prop we already have. Then, the logic for accessing the IBC status would likely live somewhere else (I'll let you find it :)).